examples/moses-pln-synergy/scm/README.md

```diff
@@ -242,7 +264,118 @@ scheme@(guile-user)> (cog-bind implication-implicant-distribution-rule)
       )
    )
 ...
-scheme@(guile-user)> ;; If X takes treatment-1 then X takes compound-A
+
+### (7) Factorize the lambda in the implicand of (6). Semi-formally
+    (\x take(x, treatment-1)) and (\x contain(treatment-1, compound-A))
+    -> (\x (take(x, treatment-1) and contain(treatment-1, compound-A)))
```

Comment:
ditto -- the variables should be changed to x y and z, instead of 3 x's in a row. They're combinators.